### ISSUE TITLE
Feature: Ollama Support

### ISSUE THREAD

**Issue by orkutmuratyilmaz** [SawyerHood/draw-a-ui#52]
Hello and thanks for this beautiful repo:)

Do you have plans for supporting Open Source LLMs, such as [Ollama](https://ollama.ai/)?

Best,
Orkut

**Comment by abi** [comment]
Does Ollama support multi modal LLMs?

**Comment by orkutmuratyilmaz** [comment]
@abi, yes but it needs a better documentation right now. Please check [this issue](https://github.com/jmorganca/ollama/issues/746).

**Comment by orkutmuratyilmaz** [comment]
Ollama is now officially supporting [LLaVA](https://ollama.ai/library/llava) and [BakLLaVA](https://ollama.ai/library/bakllava) too.

**Comment by bocklucas** [comment]
#99
